apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -617,15 +621,10 @@ handle_request_('GetStatus', _Params, _Context) ->
        <<"node_revision">>              => NodeRevision,
        <<"peer_count">>                 => PeerCount,
        <<"pending_transactions_count">> => PendingTxsCount,
-       <<"network_id">>                 => aec_governance:get_network_id()}};
-
-handle_request_('GetPeerKey', _Req, _Context) ->
-    case aehttp_logic:peer_pubkey() of
-        {ok, PeerKey} ->
-            {200, [], #{pub_key => aeser_api_encoder:encode(peer_pubkey, PeerKey)}};
-        {error, key_not_found} ->
-            {404, [], #{reason => <<"Keys not configured">>}}
-    end;
+       <<"network_id">>                 => aec_governance:get_network_id(),
+       <<"peer_pubkey">>                => aeser_api_encoder:encode(peer_pubkey, PeerPubkey),
```

Comment:
Could you please update swagger.yml (and re-generate the json) for these 3 new fields.

Reply:
Yeah, I realized I missed it myself. Now updated and the original commits has been updated. (I Plan merge instead of squash + rebase)